lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc

```diff
@@ -658,6 +658,9 @@ function paraneue_dosomething_preprocess_taxonomy_term(&$vars) {
  */
 function paraneue_dosomething_preprocess_user_profile(&$vars) {
   global $base_url;
+  global $user;
+  $language = $user->language;
```

Comment:
You can use `dosomething_helpers_get_current_language_code()` instead if you want to.